docs/content/2.deploy/providers/alwaysdata.md

```diff
@@ -0,0 +1,39 @@
+# alwaysdata
+
+Deploy Nitro apps to alwaysdata.
+
+**Preset:** `alwaysdata` ([switch to this preset](/deploy/#changing-the-deployment-preset))
+
+Nitro supports deploying on the [alwaysdata](https://www.alwaysdata.com/en/) platform with minimal configuration.
+
+## Set up application
+
+### Pre-requisites
+
+1. [Register a new profile](https://www.alwaysdata.com/en/register/) on alwaysdata platform if you don't have one.
+
+2. Get a free 100Mb plan to host your app.
+
+*Note:* Keep in mind your *account name* will be used to provide you a default URL in the form of `account_name.alwaysdata.net`, so choose it wisely. You can also link your existing domains to your account later or register as many accounts under your profile as you need.
+
+### Local deployment
+
+1. Build your project locally with `NITRO_PRESET=alwaysdata npm run build`
+
+2. [Upload your app](https://help.alwaysdata.com/en/remote-access/) to your account in its own directory (e.g. `$HOME/www/my-app`). You can use any protocol you prefer (SSH/FTP/WebDAV…); just be careful to transfer the built `.output` directory and its subdirectories. E.g. `scp -r my-app [account_name]@ssh-[account_name].alwaysdata.net:~/www`.
```

Comment:
For this in custom preset you made we can use `commands: { deploy: `scp -r ./ [account].alwaysdaa.net:~/www` }` so we show users command in the end.

Reply:
Oh sweet! Thanks!